api/src/main/java/org/openmrs/Person.java

```diff
@@ -325,6 +327,22 @@ public void setCauseOfDeath(Concept causeOfDeath) {
 		this.causeOfDeath = causeOfDeath;
 	}
 	
+	/**
```

Comment:
Can you also put a method description?

Reply:
Can you verify if the method description is sufficient now?